docs/content/dagster-plus/managing-deployments/branch-deployments.mdx

```diff
@@ -180,6 +174,47 @@ There are currently two ways to set up Branch Deployments for Dagster+. In the t
         </ul>
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Serverless setup</strong>
+      </td>
+      <td>
+        Not required if the Dagster+ GitHub app is used to{" "}
+        <a href="/dagster-plus/managing-deployments/code-locations">
+          create a code location
+        </a>
+      </td>
+      <td>
+        Not required if the Dagster+ Gitlab app is used to{" "}
+        <a href="/dagster-plus/managing-deployments/code-locations">
+          create a code location
+        </a>
+      </td>
+      <td>TODO</td>
```

Comment:
Will update once we figure out how this is supposed to work for Serverless, if at all